src/core/grid_based_algorithms/lbgpu_cuda.cu

```diff
@@ -97,11 +97,8 @@ LB_node_force_density_gpu node_f = {
 static float *lb_boundary_force = nullptr;
 #endif
 
-/** @name pointers for additional cuda check flag */
-/**@{*/
-static int *gpu_check = nullptr;
-static int *h_gpu_check = nullptr;
-/**@}*/
+/** @brief Whether LB GPU was initialized */
+static int *device_gpu_lb_initialized = nullptr;
```

Comment:
why is this an int pointer if it only signals whether LB GPU is initialized? Consider using a ```bool``` type here.